package.json

```diff
@@ -11,14 +11,14 @@
     "test": "tests"
   },
   "scripts": {
-    "build": "ember build --environment=production",
+    "build": "TAILWIND_MODE=build ember build --environment=production",
```

Comment:
What's this do? And how is it different from being omitted?
(Since the deployed site has styles)

Reply:
It's in their documentation. https://github.com/tailwindlabs/tailwindcss-jit#getting-started

> Make sure you set NODE_ENV=development if you are running a watcher, or Tailwind won't watch your template files for changes. Set NODE_ENV=production for one-off builds.
> 
> If you want to control whether Tailwind watches files or not more explicitly, set TAILWIND_MODE=watch or TAILWIND_MODE=build to override the default NODE_ENV-based behavior.
> 
> For example if you want to do one-off builds with NODE_ENV=development, explicitly set TAILWIND_MODE=build so that Tailwind knows you are just doing a one-off build and doesn't hang.

Since the default task doesn't have `NODE_ENV=production` and uses `--environment=production` from Ember CLI; it would make more sense for Tailwind to read the `TAILWIND_MODE=build` variable.

Reply:
Makes sense to me! Thanks!

Reply:
Anytime 👍